ios/Helium.xcodeproj/project.pbxproj

```diff
@@ -817,6 +817,7 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = Helium/Helium.entitlements;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_TEAM = PVM9KJZ2AD;
```

Comment:
Is this the helium development team?

Reply:
Yeah, don't know why that got changed, but it should be fine.